certora/confs/SafeWebAuthnSignerFactory.conf

```diff
@@ -1,16 +1,17 @@
 {
```

Comment:
We should also add a sanity check:
`"rule_sanity": "basic",`

Reply:
Added thanks!